### PR TITLE
GraknStateStorage remove synchronized

### PR DESCRIPTION
**Pull request by alexandraorth**

### CODE REVIEW

**Comment by fppt** [comment]
I am doing this in another PR so it should be safe to remove the synchronized lock :)

**Comment by alexandraorth** [comment]
You are also removing the synchronized lock in another PR? Not having it used to cause a ton of bugs, so I am a bit confused as to why Travis is passing without it.

**Comment by fppt** [comment]
I think it's because the thread safety of mutating the system keyspace graph has been improved under the hood. So less issues at the top.

File: grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java [hunk 1]
```diff
@@ -154,7 +154,7 @@ static void startEmbeddedCassandra() {
     static void hideLogs() {
         Logger logger = (Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
         logger.setLevel(Level.OFF);
-        org.apache.log4j.Logger.getRootLogger().setLevel(org.apache.log4j.Level.ERROR);
+//        org.apache.log4j.Logger.getRootLogger().setLevel(org.apache.log4j.Level.ERROR);
```
Comment (fppt) [inline grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java]: Commented code makes me sad.

Comment (aelred) [inline grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java]: and me!

**Pull request merged**